CMakeLists.txt

```diff
@@ -20,88 +19,90 @@ include(CheckCXXStandardSupport)
 include(BlueGenePortability)
 
 if(NOT COMPILER_SUPPORTS_CXX11)
-    message(FATAL_ERROR "HighFive version >=2.0 requires c++ standard >= c++11")
+  message(FATAL_ERROR "HighFive version >= 2.0 requires c++ standard >= c++11")
 endif()
 
-
-option(HIGHFIVE_UNIT_TESTS "enable unit tests" TRUE)
-option(USE_BOOST "enable Boost Support" TRUE)
-option(USE_EIGEN "enable Eigen testing" FALSE)
-option(USE_XTENSOR "enable xtensor testing" FALSE)
+option(HIGHFIVE_UNIT_TESTS "Enable unit tests" TRUE)
+option(USE_BOOST "Enable Boost Support" TRUE)
+option(USE_EIGEN "Enable Eigen testing" FALSE)
+option(USE_XTENSOR "Enable xtensor testing" FALSE)
 option(HIGHFIVE_EXAMPLES "Compile examples" TRUE)
 option(HIGHFIVE_PARALLEL_HDF5 "Enable Parallel HDF5 support" FALSE)
 
+if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
```

Comment:
-> `if (NOT...`
Btw, can't we put the next if here with an `OR`?

Reply:
Done